### PR TITLE
[1.13] Bump dcos-diagnostics

### PR DESCRIPTION
**Pull request by janisz**
## High-level description

Backports: https://github.com/dcos/dcos/pull/5704
---

Bump dcos-diagnostics

In https://github.com/dcos/dcos-diagnostics/commit/43298bfb1c5af73a07f233bcd8cac757d7bd7514#diff-0ee9de7b9d9927cce9fb53251662d72aR102 we removed `job_ended`, `duration`, `errors` fields from bundle status when they are empty. This commit removes checking if bundle status contains that fields.

## Corresponding DC/OS tickets (required)

  - [DCOS_OSS-5254](https://jira.mesosphere.com/browse/DCOS_OSS-5254)
  - [DCOS_OSS-5256](https://jira.mesosphere.com/browse/DCOS_OSS-5256)
  - [DCOS_OSS-5241](https://jira.mesosphere.com/browse/DCOS_OSS-5241)
  - [DCOS_OSS-5261](https://jira.mesosphere.com/browse/DCOS_OSS-5261)
  - [DCOS_OSS-5255](https://jira.mesosphere.com/browse/DCOS_OSS-5255)
  - [DCOS-5104](https://jira.mesosphere.com/browse/DCOS-5104)

# Related tickets (optional)

  - [DCOS-ID](https://jira.mesosphere.com/browse/DCOS-<number>) JIRA title / short description.

## Checklist for component/package updates:

If you are changing components or packages in DC/OS (e.g. you are bumping the sha or ref of anything underneath `packages`), then in addition to the above please also include:

  - [x] Change log from the last version integrated (this should be a link to commits for easy verification and review): [example](https://github.com/dcos/dcos-diagnostics/compare/f741a79bac1b4841f5c33e12e2dc99c4f1e9d1f5...9d32a1a77d95cb42573af29393019623854c6a17)
  - [ ] Included a test which will fail if code is reverted but test is not. If there is no test please explain.
  - [ ] Test Results: [link to CI job test results for component]
  - [ ] Code Coverage (if available): [link to code coverage report]

### CODE REVIEW

**Comment by janisz** [comment]
@mesosphere-mergebot label Ready For Review

**Comment by janisz** [comment]
:wave: @drewvanstone @br-lewis @makkes  Can you review, please?

**Comment by janisz** [comment]
@mesosphere-mergebot request-review @drewvanstone @br-lewis @makkes

File: packages/dcos-integration-test/extra/test_dcos_diagnostics.py [hunk 1]
```diff
@@ -348,16 +348,13 @@ def test_dcos_diagnostics_bundle_create_download_delete(dcos_api_session):
...
-    required_status_fields = ['is_running', 'status', 'errors', 'last_bundle_dir', 'job_started', 'job_ended',
-                              'job_duration', 'diagnostics_bundle_dir', 'diagnostics_job_timeout_min',
+    required_status_fields = ['is_running', 'status', 'last_bundle_dir', 'job_started',
+                              'diagnostics_bundle_dir', 'diagnostics_job_timeout_min',
                               'journald_logs_since_hours', 'diagnostics_job_get_since_url_timeout_min',
                               'command_exec_timeout_sec', 'diagnostics_partition_disk_usage_percent',
                               'job_progress_percentage']
 
     for _, properties in diagnostics_bundle_status.items():
-        assert len(properties) == len(required_status_fields), 'response must have the following fields: {}'.format(
```
Comment (drewvanstone) [inline packages/dcos-integration-test/extra/test_dcos_diagnostics.py]: Again, just curious why this is being removed?

Comment (janisz) [reply packages/dcos-integration-test/extra/test_dcos_diagnostics.py]: https://github.com/dcos/dcos/pull/5704#discussion_r292880694

**Pull request merged**